src/components/Table/components/Body/Body.tsx

```diff
@@ -24,6 +25,7 @@ export const Body = <Item extends ItemBase>({
         select={select}
         handleRefresh={handleRefresh}
         externalBorder={externalBorder}
+        hasStyledFirstRow={index === 0 && hasStyledFirstRow}
```

Comment:
BodyTr to już jest row, nazwa propsa dla samego wiersza "hasStyledFirstRow" jest trochę myląca, zmieniłbym na coś innego. W sensie - dla propa dla tabeli to spoko nazwa, dla samego wiersza nope ;D